parm/config/gefs/config.base

```diff
@@ -287,10 +287,17 @@ export DO_MERGENSST="NO"
 export NMEM_ENS=@NMEM_ENS@
 
 # set default member number memdir for control
-# this will be overwritten for the perturbed members
+# this will be overwridden for the perturbed members
 export ENSMEM=${ENSMEM:-"000"}
 export MEMDIR="mem${ENSMEM}"
 
+# cellular automata
+if (( ENSMEM == 0 )); then
+    export DO_CA="YES"
+else
+    export DO_CA="YES"
```

Comment:
Should one of these be `NO`?

Reply:
Nope, at least not by default right now.